test/functional/apps/context/_date_nanos_custom_timestamp.js

```diff
@@ -38,6 +38,7 @@ export default function ({ getService, getPageObjects }) {
       await kibanaServer.uiSettings.update({
         'context:defaultSize': `${TEST_DEFAULT_CONTEXT_SIZE}`,
         'context:step': `${TEST_STEP_SIZE}`,
+        'discover:searchFieldsFromSource': true,
```

Comment:
Sorting is broken in Discover when using fields API with this custom date format, so this test will fail if `discover:searchFieldsFromSource` is not set. I need to look into why that happens, but I don't think it should be a blocker for this PR.

Reply:
@majagrubic Agreed that it shouldn't be a blocker if the default value of `SEARCH_FIELDS_FROM_SOURCE` is `true`, but if  the default `SEARCH_FIELDS_FROM_SOURCE` is `false` I would hesitate to merge, since CI would be green, but the it would be broken for the user, he would need to actively set to `true` to fix it

Reply:
will have a look here

Reply:
as discussed, this should be fixed, once `date_nanos` formatting is fixed , which is the reason why this PR would fails when `searchFieldsFromSource` is set to `false`

Reply:
<img width="1441" alt="Banners_and_Alerts_und_Discover_-_Elastic" src="https://user-images.githubusercontent.com/463851/104305205-b021d180-54cc-11eb-8b92-d93628a129e3.png">

Reply:
https://github.com/elastic/elasticsearch/issues/67063